### ISSUE TITLE
Regressions with 2.6.5

### ISSUE THREAD

**Issue by BurhanDanger** [MuntashirAkon/AppManager#638]
- [x] I know what my device, OS and App Manager versions are
- [ ] I know how to take logs
- [ ] I know how to reproduce the issue which may not be specific to my device

**Describe the bug**
Video attached. 

**Expected behavior**
Not crash in those case. 2.6.4 doesn't crash in this case. or show misinformation.

**Media**
https://user-images.githubusercontent.com/25033866/143870896-f342cc9a-360c-411b-962e-580e6b29074d.mp4

**Crash logs**
```
java.lang.IllegalArgumentException: ApkFile not found for key 404879949
    at io.github.muntashirakon.AppManager.apk.ApkFile.getInstance(ApkFile.java:89)
    at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.startActivityForSplit(AppInfoFragment.java:1115)
    at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.lambda$setHorizontalActions$81$AppInfoFragment(AppInfoFragment.java:1015)
    at io.github.muntashirakon.AppManager.details.info.-$$Lambda$AppInfoFragment$_KYe0pTmPF1_jNCQwBAkKnzyCzU.onClick(Unknown Source:2)
    at android.view.View.performClick(View.java:7448)
    at android.view.View.performClickInternal(View.java:7425)
    at android.view.View.access$3600(View.java:810)
    at android.view.View$PerformClick.run(View.java:28305)
    at android.os.Handler.handleCallback(Handler.java:938)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loop(Looper.java:223)
    at android.app.ActivityThread.main(ActivityThread.java:7671)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:594)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)

Device Info:
App version: 2.6.5
App version code: 396
Android build version: eng.noob.20211129.024505
Android release version: 11
Android SDK version: 30
Android build ID: lineage_miatoll-userdebug 11 RD2A.211001.002 eng.noob.20211129.024505 release-keys
Device brand: Redmi
Device manufacturer: Xiaomi
Device name: curtana
Device model: Redmi Note 9 Pro
Device product name: lineage_miatoll
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-GB
In-App Language: auto
Mode: root
```

**Comment by MuntashirAkon** [comment]
Seems to be a backporting issue. I'll release an emergency patch later on.

**Comment by MuntashirAkon** [comment]
Fixed in 99da0486
